demo/src/pages/Workout.tsx

```diff
@@ -527,13 +529,64 @@ const Workout = ({ history }) => {
     isPreJoin,
   };
 
+  const [connectionDetails, setConnectionDetails] = useState() as any;
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get('https://ipapi.co/json/')
+        const { data } = response;
+        setConnectionDetails({
+          ipAddress: data?.ip,
+          location: `${data?.city}, ${data?.region_code}, ${data?.country}`
```

Comment:
Can we also save time zone? We should mostly get it from Moment. We may have a function in Utils to return the timezone.